### ISSUE TITLE
Integrated Terminal does not open after update.

### ISSUE THREAD

**Issue by lucidviews** [microsoft/vscode#159738]
Type: <b>Bug</b>

Today I upodated my VS code to version Version: 1.70.2 (Universal) and ever since I cannot open a terminal anymore. When calling the "create new terminal" command, the termial appears for a split second and closes right after. 

VS Code version: Code 1.70.2 (Universal) (e4503b30fc78200f846c62cf8091b76ff5547662, 2022-08-16T05:36:37.829Z)
OS version: Darwin arm64 21.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 4, 4|
|Memory (System)|16.00GB (0.08GB free)|
|Process Argv|--crash-reporter-id d676b824-195a-46a2-87b3-5ff782d8d3ae --crash-reporter-id d676b824-195a-46a2-87b3-5ff782d8d3ae|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-pull-request-github|Git|0.48.1
vscode-docker|ms-|1.22.1
python|ms-|2022.12.1
vscode-pylance|ms-|2022.8.50
jupyter|ms-|2022.7.1102252217
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.9
remote-containers|ms-|0.245.2
tabnine-vscode|Tab|3.6.14

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411cf:30557515
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
pylanb8912:30545647
vsclangdc:30486549
c4g48928:30535728
hb751961:30553087
dsvsc012cf:30540253
azure-dev_surveyone:30548225
i497e931:30553904

```

</details>

**Comment by gjsjohnmurray** [comment]
Does setting `"terminal.integrated.shellIntegration.enabled": "false"` help?

**Comment by lucidviews** [comment]
Unfortunately not.

**Comment by gjsjohnmurray** [comment]
1.71 is likely to ship later today. I suggest you re-test after that update lands, then report back here.

**Comment by meganrogge** [comment]
Can you share your settings.json file and also do a trace log while creating a terminal? https://github.com/microsoft/vscode/wiki/Terminal-Issues#enabling-trace-logging
/label info-needed terminal

**Comment by andrewburgess** [comment]
Not sure if this is related, but after the update (v1.71.0), I can open a new terminal, but there seems to be extra `"` marks
![image](https://user-images.githubusercontent.com/79218/188028809-bbdeb9d1-48d3-4975-b9e6-606a7659e8b3.png)

and trying to split an active terminal causes similar behavior and I see an error box with what seems to be an extra set of `"` around the directory:
![image](https://user-images.githubusercontent.com/79218/188028835-86b67412-4376-42a2-8b53-c27370d0bf82.png)

**Comment by tekumara** [comment]
Same problem in

Version: 1.71.0
Commit: 784b0177c56c607789f9638da7b6bf3230d47a8c
Date: 2022-09-01T07:25:25.516Z (20 hrs ago)
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: No

**Comment by lucidviews** [comment]
> Can you share your settings.json file and also do a trace log while creating a terminal? https://github.com/microsoft/vscode/wiki/Terminal-Issues#enabling-trace-logging /label info-needed terminal

tarce log:
[2022-09-02 09:19:10.420] [renderer17] [critical] Extension 'tabnine.tabnine-vscode' wants API proposal 'inlineCompletions' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
[2022-09-02 09:19:10.470] [renderer17] [error] [tht13.python]: 'configuration.jsonValidation.url' must be an absolute URL or start with './'  to reference schemas located in the extension.

settings.json:
{
    "security.workspace.trust.untrustedFiles": "open",
    "terminal.integrated.shell.windows": "C:\\WINDOWS\\System32\\cmd.exe",
    "tabnine.experimentalAutoImports": true,
    "window.zoomLevel": -1,
    "files.associations": {
        "*.rmd": "markdown"
    },
    "terminal.integrated.shellIntegration.enabled": false
}

**Comment by mtaylor769** [comment]
Same issue. You can't break the terminal with a new release, ya dopes.

**Comment by lucidviews** [comment]
Issue resolved after updating to 1.71....
